ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/editor/plugin/DependencyAnalysisSection.java

```diff
@@ -40,8 +40,9 @@ public DependencyAnalysisSection(PDEFormPage page, Composite parent, int style)
 	}
 
 	private String getFormText() {
-		boolean editable = getPage().getModel().isEditable();
-		if (getPage().getModel() instanceof IPluginModel) {
+		IBaseModel model = getPage().getModel();
+		boolean editable = model != null && model.isEditable();
+		if (model instanceof IPluginModel) {
```

Comment:
In case the model is null this method returns the text for fragments. I don't think that is intended.
If null is a valid case now that should be be handled somehow. My first suggestion would be to use the Plugin text case case the model is null and the 'type' is therefore unknown.
```suggestion
		if (model instanceof IPluginModel || model == null) {
```

Reply:
It could happen that the model is (not yet) there, in such a case one can not give a suitable hint and I return the empty string for now.